shared/nav.desktop.js

```diff
@@ -191,7 +191,7 @@ export default connect(
   ({
     router,
     config: {extendedConfig, username},
-    favorite: {publicBadge, privateBadge},
+    favorite: {publicBadge = 0, privateBadge = 0},
```

Comment:
these could be undefined, which we then add together which then turns it into a NaN which causes the nav to render constantly since NaN !== NaN